### PR TITLE
fixing issues #1262 and #766

### PR DESCRIPTION
**Pull request by manonthemat**
added github icon (fixing #1262)

search bar closes on mobile when unfocused (fixing #766)

### CODE REVIEW

File: 1.0/index.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ title: Welcome
       </a>
       <a href="https://github.com/polymer">
         <paper-button class="github" unresolved>
-          <core-icon icon="social:post-github"></core-icon> View on GitHub
+          <core-icon src="components/component-download-button/icons/github.png"></core-icon> View on GitHub
```
Comment (ebidel) [inline 1.0/index.md]: Im a little hesitant to rely on a component's image for the site. Can we just check in the log and reference it from static/images?

File: 1.0/elements/app-bar.html [hunk 2]
```diff
@@ -67,6 +67,10 @@
           this.$.input.focus();
         });
       },
+      disableSearch: function() {
+        this.classList.toggle('search-on');
```
Comment (ebidel) [inline 1.0/elements/app-bar.html]: mine as well be safe:

    this.classList.remove('search-on');

**Comment by ebidel** [comment]
Reviewed

**Comment by manonthemat** [comment]
Thanks for the suggestions. I made those changes.

**Comment by ebidel** [comment]
LGTM

**Pull request merged**